server.js

```diff
@@ -0,0 +1,116 @@
+const School = require('./School');
+const express = require('express');
+const cors = require('cors');
+const bodyParser = require('body-parser');
+
+
+let mySchool = new School();
+
+
+const port = 4000;
+
+const app = express();
+app.use(cors());
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+
+const validateClass = (req, res, next) => {
+   let name = req.body.name;
+   let teacher = req.body.teacher;
+   if (!name || !teacher || mySchool.classes[name]) {
+      res.status(400).json({
+         status: "bad request",
+         error: "Please fill out all the information or Class already exists",
+         timestamp: new Date()
+      })
+      return
+   } else {
+      next()
+   }
+}
+
+app.post("/class", validateClass, (req, res) => {
+   mySchool.addClass(req.body.name, req.body.teacher)
+   let newClass = { name: req.body.name, teacher: req.body.teacher }
+   res.status(200).json({
+      status: "success",
+      message: "Created a new class",
+      class: newClass,
+      timestamp: new Date()
+   })
+})
+
+const validateClassName = (req, res, next) => {
+   let className = req.params.className
+
+   if (!className) {
+      res.status(400).json({
+         status: "bad request",
```

Comment:
You wouldn't need a `status: bad request` since on like 48 you're already adding the status to be 400.